tests/test_launch_methods/test_jsrun/jobs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Begin LSF Directives
+#BSUB -P BIP178
+#BSUB -W 0:30
+#BSUB -nnodes 2
+#BSUB -J jsrun_test
+#BSUB -o jsrun_test.%J
+#BSUB -e jsrun_test.%J
+#BSUB -N rct-tests@googlegroups.com
+
+# CONFIG
+path=`pwd`
+d=`date +%D | sed 's/\//_/g'`
+host=`/bin/hostname`
+#dest="rct-tests@googlegroups.com"
+dest="vivek.bala@rutgers.edu"
+
+run_cmd() {
+    res_set=$1
+    omp_env=$2
+    cmd=$3
+    exp_out=$4
+    tid=$5
+
+    # Create required folders
+    rm results_$d/$tid -rf
+    mkdir results_$d/$tid
+
+    # Record resource file, omp env, and cmd in log
+    echo $res_set >> results_$d/$tid/full_output.log
+    echo $omp_env >> results_$d/$tid/full_output.log
+    echo $cmd >> results_$d/$tid/full_output.log
+
+    # Set omp env, create resource file and run cmd
+    $omp_env
+    echo $res_set >> $path/resource-file
+    cmd=`echo $cmd | sed "s#resource-file#$path/resource-file#g"`
```

Comment:
Since there is a resource file in each folder why you write it to a different file and then remove it?

Reply:
The content of the resource file is appended to the output log inside each folder. The resource file itself
is created in ``path`` and then deleted at the end of the individual test.

Reply:
Yes, but why are you doing all that, when you can just  use the `res_set` file as input?

Reply:
Fair point. Changed and tested.